### PR TITLE
Skybrowser images drag and drop

### PR DESCRIPTION
**Pull request by ylvaselling**
Change the image layer reordering interface to drag and drop. 

Use with https://github.com/OpenSpace/OpenSpace-WebGuiFrontend/pull/121

### CODE REVIEW

File: modules/skybrowser/src/wwtcommunicator.cpp [hunk 1]
```diff
@@ -290,16 +291,34 @@ glm::dvec2 WwtCommunicator::equatorialAim() const {
...
+    int currentIndex = std::distance(_selectedImages.begin(), current);
+    int difference = order - currentIndex;
 
-    // Make sure the image was found in the list
-    if (current != _selectedImages.end() && target != _selectedImages.end()) {
-        // Swap the two images
-        std::iter_swap(current, target);
+    std::deque<std::pair<int, double>> newDeque;
+
+    for (int i = 0; i < _selectedImages.size(); i++) {
```
Comment (WeirdRubberDuck) [inline modules/skybrowser/src/wwtcommunicator.cpp:302]: This should yield a warning for comparing an int (`i`) with a `size_t` (`_selectedImages.size()`), that should be fixed before merging

**Review by WeirdRubberDuck — APPROVED** [review]
Looks good to me! :) And works well apart form perfoance issue in CEF UI :) (see comment in webgui PR)

File: modules/skybrowser/src/wwtcommunicator.cpp [hunk 2]
```diff
@@ -290,16 +291,34 @@ glm::dvec2 WwtCommunicator::equatorialAim() const {
 }
 
 void WwtCommunicator::setImageOrder(int i, int order) {
+    using it = std::deque<std::pair<int, double>>::iterator;
     // Find in selected images list
-    auto current = findSelectedImage(i);
-    auto target = _selectedImages.begin() + order;
+    it current = findSelectedImage(i);
```
Comment (alexanderbock) [inline modules/skybrowser/src/wwtcommunicator.cpp:296]: Can also just do `auto it current = findSelectedImage(i);`  we allow `auto` in place of container iterators

**Review by alexanderbock — APPROVED** [review]

**Comment by ylvaselling** [comment]
This is waiting for the gui hash appear in https://data.openspaceproject.com/files/webgui/frontend/
After than it can be merged:)

**Pull request merged**